<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1122</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>0</height>
      </size>
     </property>
     <property name="lineWidth">
      <number>3</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>6</number>
     </property>
     <property name="childrenCollapsible">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaContainer" native="true">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>500</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>393</width>
            <height>835</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_image">
             <property name="title">
              <string>Image</string>
             </property>
             <layout class="QFormLayout" name="formLayout_2">
              <item row="0" column="0">
               <widget class="QLabel" name="label_transformation">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Orientation of the displayed image, does not affect the 3D image stack. Currently &lt;span style=&quot; font-style:italic;&quot;&gt;not available&lt;/span&gt; in conjunction with ROI line plots.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string>Transformation</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="comboBox_transform">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Orientation of the displayed image, does not affect the 3D image stack. Currently &lt;span style=&quot; font-style:italic;&quot;&gt;not available&lt;/span&gt; in conjunction with ROI line plots.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <item>
                 <property name="text">
                  <string>none</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>flipUD</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>flipLR</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>transpose</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>rot90</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>rot180</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>rot270</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>rot180+tr</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QCheckBox" name="checkBox_maskBelow">
                <property name="text">
                 <string>Mask below</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QDoubleSpinBox" name="doubleSpinBox_maskBelow">
                <property name="cursor">
                 <cursorShape>ArrowCursor</cursorShape>
                </property>
                <property name="locale">
                 <locale language="C" country="AnyCountry"/>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="correctionMode">
                 <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
                </property>
                <property name="showGroupSeparator" stdset="0">
                 <bool>true</bool>
                </property>
                <property name="decimals">
                 <number>1</number>
                </property>
                <property name="minimum">
                 <double>-1000000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>10000000.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>10.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QCheckBox" name="checkBox_maskAbove">
                <property name="text">
                 <string>Mask above</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QDoubleSpinBox" name="doubleSpinBox_maskAbove">
                <property name="cursor">
                 <cursorShape>ArrowCursor</cursorShape>
                </property>
                <property name="mouseTracking">
                 <bool>true</bool>
                </property>
                <property name="locale">
                 <locale language="C" country="AnyCountry"/>
                </property>
                <property name="wrapping">
                 <bool>false</bool>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="correctionMode">
                 <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
                </property>
                <property name="decimals">
                 <number>1</number>
                </property>
                <property name="minimum">
                 <double>-1000000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>10000000.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>10.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QCheckBox" name="checkBox_autoColor">
                <property name="text">
                 <string>Auto colorscale</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QCheckBox" name="checkBox_fixAspect">
                <property name="text">
                 <string>Fix aspect</string>
                </property>
               </widget>
              </item>
              <item row="7" column="1">
               <widget class="QPushButton" name="pushButton_scaleTranslate">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Scale/Translate...</string>
                </property>
               </widget>
              </item>
              <item row="7" column="0">
               <widget class="QPushButton" name="pushButton_calibrate">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Calibrate...</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="Line" name="line">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_background">
             <property name="title">
              <string>Background</string>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QCheckBox" name="checkBox_subtract">
                <property name="text">
                 <string>Subtract</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLineEdit" name="lineEdit_subtract">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>200</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="cursor">
                 <cursorShape>IBeamCursor</cursorShape>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QToolButton" name="toolButton_subtract">
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QCheckBox" name="checkBox_divide">
                <property name="text">
                 <string>Divide</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLineEdit" name="lineEdit_divide">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>200</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QToolButton" name="toolButton_divide">
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_range">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="title">
              <string>Range</string>
             </property>
             <layout class="QFormLayout" name="formLayout">
              <item row="4" column="1">
               <widget class="QComboBox" name="comboBox_filter">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <item>
                 <property name="text">
                  <string>mean</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>max</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_rangeFilter">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Filter</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QCheckBox" name="checkBox_useRange">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If used several images can be filtered with &lt;span style=&quot; font-style:italic;&quot;&gt;mean&lt;/span&gt; or &lt;span style=&quot; font-style:italic;&quot;&gt;max&lt;/span&gt; filters.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string>Use range</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_rangeStart">
                <property name="text">
                 <string>From</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_rangeFinish">
                <property name="text">
                 <string>To</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QSpinBox" name="spinBox_rangeStart">
                <property name="cursor">
                 <cursorShape>ArrowCursor</cursorShape>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="correctionMode">
                 <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
                </property>
                <property name="maximum">
                 <number>100000</number>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QSpinBox" name="spinBox_rangeFinish">
                <property name="cursor">
                 <cursorShape>ArrowCursor</cursorShape>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="correctionMode">
                 <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
                </property>
                <property name="maximum">
                 <number>100000</number>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QCheckBox" name="checkBox_fullStack">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Filtering the full image stack; range selectors disabled&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string>Full stack</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_linePlots">
             <property name="title">
              <string>Line plots</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QWidget" name="widget_2" native="true">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QLabel" name="label_linePlots">
                   <property name="text">
                    <string>Line plots</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="comboBox_ROI">
                   <item>
                    <property name="text">
                     <string>none</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>ROI</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Line cut</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="spinBox_ROI">
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                   <property name="minimum">
                    <number>1</number>
                   </property>
                   <property name="maximum">
                    <number>10</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QScrollArea" name="scrollArea_2">
                <property name="widgetResizable">
                 <bool>true</bool>
                </property>
                <widget class="QWidget" name="scrollAreaWidgetContents_2">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>335</width>
                   <height>70</height>
                  </rect>
                 </property>
                 <layout class="QFormLayout" name="formLayout_4">
                  <item row="0" column="0">
                   <widget class="QPushButton" name="pushButton">
                    <property name="contextMenuPolicy">
                     <enum>Qt::CustomContextMenu</enum>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="checkable">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLabel" name="label_2">
                    <property name="text">
                     <string>Line cut 1</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QPushButton" name="pushButton_2">
                    <property name="text">
                     <string>+</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="plotArea" name="plotArea" native="true">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>500</height>
       </size>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>plotArea</class>
   <extends>QWidget</extends>
   <header location="global">plotarea</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
